lang/main.json

```diff
@@ -643,6 +643,9 @@
     },
     "calendarSync": {
         "addMeetingURL": "Add a meeting link",
+        "confirmAddLink": "Do you want to add a Jitsi link to this event?",
+        "confirmAddLinkTitle": "Calendar",
+        "today": "Today",
```

Comment:
The Today string already exists below.

Reply:
Oh, it must be some merge issue. Thanks!